src/data/AudioService.js

```diff
@@ -20,27 +22,34 @@ const AudioService = {
   /**
    * Retrieve audio details by Id number.
    * SSR-called
+   * @param {{id: string}} params
+   * @return {Promise<{data: Object}>}
```

Comment:
Fun fact, `Object`, `object` and `{}` all have different meanings in TypeScript. https://stackoverflow.com/questions/49464634/difference-between-object-and-object-in-typescript

It seems like here we actually might want `any` or `unknown` if we don't have a better type for it?

Reply:
Thank you for this comment, I didn't know! Interesting read :)